sys-libs/timezone-data/timezone-data-2021a.ebuild

```diff
@@ -150,28 +150,29 @@ configure_tz_data() {
 	fi
```

Comment:
Can you update a commit message to clarify the fate of `/etc/timezone`? Now it claims it will also be a symlink. While you are deleting it.

Reply:
I reworded the last paragraph to make this more clear.